pages/sections/landing.tsx

```diff
@@ -10,7 +10,215 @@ import SignupCounter from '@/components/SignupCounter';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import Image from 'next/image';
-import Skyline from '../../public/assets/landing.svg';
```

Comment:
scope creepin again, but can u do the same thing for dark mode 😓. since ur a master at the landing page, thut u shud do it :)